### PR TITLE
Displays user info from Codeforces API and few UI adjustments

### PR DESCRIPTION
**Pull request by rajkumaar23**
Related to #20 
Signed-off-by: Rajkumar S <rajkumaar2304@gmail.com>

### CODE REVIEW

File: app/src/main/java/com/example/android/codeforces/HomeFeedActivity.java [hunk 1]
```diff
@@ -73,11 +86,59 @@ public void onClick(View v) {
 
         if (getIntent() != null) {
             preferredHandle = getIntent().getExtras().getString(preferredHandleKey);
+            getSupportActionBar().setSubtitle("Handle : "+preferredHandle);
```
Comment (Cyril-WIRTH) [inline app/src/main/java/com/example/android/codeforces/HomeFeedActivity.java]:
```suggestion
            getSupportActionBar().setSubtitle(String.format("Handle : %s",preferredHandle));
```

Comment (rajkumaar23) [reply app/src/main/java/com/example/android/codeforces/HomeFeedActivity.java]: Done ✅

**Comment by rajkumaar23** [comment]
@Chromicle would you review my pull request ?

File: app/src/main/java/com/example/android/codeforces/HomeFeedActivity.java (hunk 1 above)
Comment (Chromicle) [reply app/src/main/java/com/example/android/codeforces/HomeFeedActivity.java]: > 

Thank you for reviewing

**Comment by Chromicle** [comment]
Can you please add the screen shor or recording so, I can review it easily

**Comment by rajkumaar23** [comment]
> Can you please add the screen shor or recording so, I can review it easily

Here you go! ^_^

- ### Contest History Activity
image 

- ### User Profile Activity
image

**Comment by Chromicle** [comment]
Hey, I am also a student from amrita university of amritapuri campus 
You can contribute to our club [repo](https://github.com/amfoss/TempleApp)
You can find great issues there please feel free to add any issues there

**Comment by rajkumaar23** [comment]
> Hey, I am also a student from amrita university of amritapuri campus
> You can contribute to our club [repo](https://github.com/amfoss/TempleApp)
> You can find great issues there please feel free to add any issues there

Yeah sure :) 
And, I'm the developer of `Amrita Repository` app. In case you hadn't heard about it, check it out at http://bit.ly/amritarepo. 
> PS : Will be useful for cbe students only anyways.

**Comment by Chromicle** [comment]
Yeah, I am one of users of it, you designed it very cool
I also made some basic one for `amrita Resource` but I dont have good source for it
You made for question papers I want to made it for class notes

**Pull request merged**